code/datums/special_traits/traits/traits.dm

```diff
@@ -289,33 +317,37 @@
 /datum/special_trait/gourmand
 	name = "Gourmand"
 	greet_text = span_notice("I can eat even the most spoiled, raw, or toxic food and water as if they were delicacies..")
-	weight = 100
+	weight = 200
 
 /datum/special_trait/gourmand/on_apply(mob/living/carbon/human/character, silent)
 	ADD_TRAIT(character, TRAIT_NASTY_EATER, "[type]")
+	character.change_stat("endurance", 4) // If you can endure drinking sewage...
```

Comment:
Shouldn't this be Constitution rather than Endurance? Health, rather than Stamina.

Reply:
No, they aren't HEALTHY from consuming rotten trash. They have have the STAMINA to stomach it.

Reply:
Noonononono stomaching it is definitely CON,END is things like enduring pain and being able to keep on running after a while without running out of breath, CON is things like not passing out from poisons and tranquilizers, eating food without throwing up and not getting knocked out after a couple of punches to the head.